www/templates/intro/consent-text.html

```diff
@@ -1,17 +1,117 @@
 <div class="consent-title">
-<center><h4>E-Mission: Data driven carbon emission reduction</h4></center>
+<center><h4>ONLINE PARTICIPANT INFORMATION STATEMENT
+  <br>Collecting personal travel behaviour data with location-aware mobile
+  application technologies</br></h4></center>
+<center></center>
 </div>
 <div class="consent-text">
-<h3>Introduction and Purpose</h3>
-This app is part of a travel behavior research platform developed at the
-University of California, Berkeley. The platform development is being led by <a
-href="http://www.cs.berkeley.edu/~shankari">K. Shankari</a> under the
-supervision of <a href="http://www.cs.berkeley.edu/~culler/">Prof. David
-Culler</a>. Our goal is to collect information from users about travel behavior
-across all modes, and use the aggregate information for better transportation
-infrastructure planning.
-
-<h3>What we collect</h3>
+<h3>What is the research study about?</h3>
+
+<div style="text-align: justify;">
+  You are invited to participate in this research study. The aim of this research study is to collect information about the personal travel behaviour of residents in all metropolitan areas across Australia. Information about people's day-to-day travel behaviour is critical to predict future travel demand and infrastructure needs. This information is even more important during a rare event such as the current virus pandemic. Having this data will allow transport services to be better planned for the most vulnerable people in our society, when the next pandemic comes.
+</div>
+
+<h3>Who is conducting this research?</h3>
+<div style="text-align: justify;">
+The study is being carried out by Assoc Prof Taha H. Rashidi and Amarin Siripanich, Research Centre for Integrated Transport Innovation, UNSW Australia.<br>Research Funder: This research is being funded by the Research Centre for Transport Integrated Innovation at UNSW Australia.</br> 
+</div>
+<h3>Inclusion criteria</h3>
+<div style="text-align: justify;">
+Before you decide to participate, we need to ensure that you meet the inclusion criteria of this research study. You can participate if the following criteria apply to you:
+</div>
+<ul>
+  <li>You are eighteen years old or older.</li>
+  <li>You reside in a metropolitan area of Australia.</li>
+  <li>You own a smartphone running Android (compatibility varies with device) or Apple iOS 9 (or newer).</li>
+  <li>You have a Google/Gmail account.*</li>
```

Comment:
Do you need this given the switch to randomized token generation?

Reply:
@asiripanich

Reply:
Updated in this .merge commit https://github.com/e-mission/e-mission-phone/pull/798/commits/b1b8bfdca11393bd4ac91a2cdb128993a310cc41